src/gas/GasFeeController.ts

```diff
@@ -18,20 +18,29 @@ import {
 
 export type unknownString = 'unknown';
 
+export type FeeMarketEstimateType = 'fee-market';
```

Comment:
It might be helpful to add a brief comment here that explains that the `FeeMarketEstimateType` is the type used for EIP1559 compatible estimates

Reply:
@danjm added comments to each type. de81a41